libr/anal/p/anal_arm_cs.c

```diff
@@ -3153,6 +3320,11 @@ static int analop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len, RAn
 		return haa;
 	}
 
+	int haa1 = hack_arm_anal(a, op, buf, len);
```

Comment:
`hack_arm_anal` can probably, and should probably be merged into `hackyArmAnal`.

Reply:
add a space before (. yeah, will be good to have all this special instructions stuff in a single function, called from asm and anal (will be squashed into r_arch at some point when ready)